lib/rdoc/code_object/method_attr.rb

```diff
@@ -415,4 +415,16 @@ def to_s # :nodoc:
     end
   end
 
+  def name_codepoint_range # :nodoc:
+    case name.codepoints[0]
```

Comment:
TIL `codepoints`

Reply:
`case name.ord`

Reply:
Thank you. See https://github.com/ruby/rdoc/pull/1220